quarkus/pom.xml

```diff
@@ -36,8 +36,8 @@
         <jackson.version>2.12.1</jackson.version>
         <jackson.databind.version>${jackson.version}</jackson.databind.version>
         <hibernate.version>5.4.29.Final</hibernate.version>
-        <mysql-connector-java.version>8.0.24</mysql-connector-java.version>
-        <postgresql.version>42.2.20</postgresql.version>
+        <mysql-connector-java.version>8.0.23</mysql-connector-java.version>
+        <postgresql.driver.version>42.2.19</postgresql.driver.version>
```

Comment:
Why there is different version than https://github.com/keycloak/keycloak/pull/8034/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R142

Reply:
My suggestion would be to keep latest versions for quarkus.

Reply:
@pedroigor Would you like to keep separate properties for JDBC driver versions in `quarkus/pom.xml` or is it ok to reuse/inherit those from the root `pom.xml`?

Reply:
@tkyjovsk Please keep the versions independent for the time being, and do not change those.

Reply:
@hmlnarik Just to clarify, should I keep the original property names in `quarkus/pom.xml` or just the versions?

Note that I renamed the properties in the root pom to distinguish versions of DB and JDBC driver. Maybe it would be good to rename them in `quarkus/pom.xml` as well, to be consistent across project, but keep separate versions.

Reply:
Thanks for asking. Just the versions, the property names need to be consistent across codebase

Reply:
> @pedroigor Would you like to keep separate properties for JDBC driver versions in `quarkus/pom.xml` or is it ok to reuse/inherit those from the root `pom.xml`?

Right now, we are using versions from Quarkus. Perhaps we should keep them separate for now until we finish the quarkus dist.